Zend/tests/type_declarations/literal_types/false_no_coercion.phpt

```diff
@@ -0,0 +1,28 @@
+--TEST--
+No coercion should be applied to type false
+--FILE--
+<?php
+
+function test(false $v) { var_dump($v); }
```

Comment:
I'd be curious about inheritance test where overridden methods accept a `false`/`true`/`null` param, while the overriding ones accept a `bool`/`bool`/`something|null` param, respectively. This kind of variance is supported, right?

Reply:
Ping :)

Reply:
It should yes, I'll add a test just to make sure of it

Reply:
Do the new test clarify the variance? Or should I add more?

Reply:
Yes, it does for return types. Would it be worth to add similar tests for params?